src/routes/authentication.js

```diff
@@ -113,11 +115,274 @@ authenticationRouter.post('/signup', validateUserSignup, Agentsignup);
  *         description: Internal server error
  */
 authenticationRouter.post('/login', agentLogin);
+
+/**
+ * @swagger
+ * definitions:
+ *   Properties:
+ *     properties:
+ *       image:
+ *         type: string
+ *       title:
+ *         type: string
+ *       price:
+ *          type: string
+ *       status:
+ *         type: string
+ *       address:
+ *         type: string
+ *       city:
+ *         type: string
+ *       neigbourhood:
+ *         type: string
+ *       LGA:
+ *         type: string
+ *       ZIP:
+ *         type: string
+ *       numberOfBaths:
+ *         type: string
+ *       numberOfBed:
+ *         type: string
+ *       landSize: string
+ *     example: {
+ *       "image": game.png,
+ *       "title": "duplex",
+ *       "price": "200000000",
+ *       "address": "abudu abayomi street",
+ *       "city": "lagos",
+ *       "neigbourhood": "ojodu berger",
+ *       "LGA": "ikeja",
+ *       "ZIP": "11011",
+ *       "numberOfBaths": "3",
+ *       "numberofbed": "4",
+ *       "landSize": "4000"
+ *     }
+ */
+
+/**
+ * @swagger
+ * /agent/properties:
+ *   post:
+ *     tags:
+ *       - Property Details
+ *     description: Add a new property to the database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: properties
+ *         description: property object
+ *         in: body
+ *         required: true
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *       - name: Authorization
+ *         in: header
+ *         description: an authentication header
+ *         required: true
+ *         type: string
+ *     responses:
+ *       201:
+ *         description: property uploaded successfully
+ *       400:
+ *         description: Bad input supplied
+ *       500:
+ *         description: Internal server error
+ */
+authenticationRouter.post(
+  '/agent/properties',
+  checkToken,
+  validatePropertyInput,
+  createProperty
+);
+
+/**
+ * @swagger
+ * /properties:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *           type: object
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *         example: {
```

Comment:
When you retrieve all properties, do you return back a single property? This example is supposed to be an array of property that mimicks what your response payload looks like.